It is not GNOME business to override distribution choices for default
monospace and document font.
---
 schemas/org.gnome.desktop.interface.gschema.xml.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/schemas/org.gnome.desktop.interface.gschema.xml.in b/schemas/org.gnome.desktop.interface.gschema.xml.in
--- a/schemas/org.gnome.desktop.interface.gschema.xml.in
+++ b/schemas/org.gnome.desktop.interface.gschema.xml.in
@@ -156,14 +156,14 @@
       </description>
     </key>
     <key name="document-font-name" type="s">
-      <default>'Adwaita Sans 11'</default>
+      <default>'Sans 11'</default>
       <summary>Document font</summary>
       <description>
         Name of the default font used for reading documents.
       </description>
     </key>
     <key name="monospace-font-name" type="s">
-      <default>'Adwaita Mono 11'</default>
+      <default>'Monospace 11'</default>
       <summary>Monospace font</summary>
       <description>
         Name of a monospaced (fixed-width) font for use in locations like
--- 
2.20.1
